plugins/obs-webrtc/whip-output.cpp

```diff
@@ -428,7 +427,19 @@ bool WHIPOutput::Connect()
 	long redirect_count = 0;
 	curl_easy_getinfo(c, CURLINFO_REDIRECT_COUNT, &redirect_count);
 
-	if (location_headers.size() < static_cast<size_t>(redirect_count) + 1) {
+	std::string last_location_header = "";
+	size_t location_header_count = 0;
+	for (auto &http_header : http_headers) {
+		auto value = value_for_header("location", http_header);
+		if (value == "") {
```

Comment:
Nitpicks:
- `value == ""` -> `value.empty()`
- No brackets since it's just a single statement